addons/linecompass/stringtable.xml

```diff
@@ -5,74 +5,92 @@
+                <German>Linienkompass</German>
             </Key>
             <Key ID="STR_dui_linecompass_enabled">
                 <English>Enabled</English>
                 <French>Activé</French>
+                <German>Aktiviert</German>
             </Key>
             <Key ID="STR_dui_linecompass_compass_required">
                 <English>Show Only When Compass is Available</English>
                 <French>Afficher uniquement lorsque le compas est disponible</French>
+                <German>Nur zeigen, wenn der Kompass verfügbar ist</German>
             </Key>
             <Key ID="STR_dui_linecompass_draw_directions">
                 <English>Directions Drawn</English>
                 <French>Directions indiquées</French>
+                <German>Angezeigte Richtungen</German>
             </Key>
             <Key ID="STR_dui_linecompass_icon_outline">
                 <English>Icon Outline</English>
                 <French>Contour des icônes</French>
+                <German>Icon Umriss</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_icon_color">
                 <English>Default Icon Color</English>
                 <French>Couleur de l'icône par défaut</French>
+                <German>Standardfarbe des Icons</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_icon_color_desc">
                 <English>Default color for icons without fire team assignment</English>
                 <French>Couleur par défaut des icônes sans affectation à une équipe</French>
+                <German>Standardfarbe für Icons ohne Zuweisung zu einem Feuerteam</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_waypoint_color">
                 <English>Default Waypoint Color</English>
                 <French>Couleur par défaut du point de passage</French>
+                <German>Standardfarbe der Wegpunkte</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_waypoint_color_desc">
                 <English>Waypoint given by squad leader or mission</English>
                 <French>Point de repère donné par le chef d'équipe ou la mission</French>
+                <German>Wegpunkt, gegeben durch Gruppenführer oder Mission</German>
             </Key>
             <Key ID="STR_dui_linecompass_custom_waypoint_color">
                 <English>Custom Waypoint Color</English>
                 <French>Couleur personnalisée du point de passage</French>
+                <German>Personalisierte Farbe für Wegpunte</German>
             </Key>
             <Key ID="STR_dui_linecompass_custom_waypoint_color_desc">
                 <English>Waypoint created when shift+click is on the map is used</English>
-                <French>Création d'un point de passage lorsque shift+click est utilisé sur la carte</French>
+                <French>Création d'un point de passage lorque shift+click est utilisé sur la carte</French>
```

Comment:
Kurze Frage, warum wurde hier `lorsque` zu `lorque` geändert?

Reply:
Really up to you, either you wait until that one is merged and you rebase/merge master into your branch and update. Or you make a followup PR.
what ever you prefer.

Reply:
I like my PRs clean and not unnecessarily small :) so I'll wait for #245 to merge and rebase then. I will also fix this line in that rebase, so please keep this review open :).

Reply:
The pr has been merged.